.concourse/tasks/upgrade.sh

```diff
@@ -1,10 +1,40 @@
 #!/bin/bash
 
+# Notes
+# - Parameters:
+#   - GKE_KEY
+#   - GKE_PROJECT
+#   - GKE_ZONE
+#
+# - Variables used by catapult:
+#   - GKE_CLUSTER_NAME
+#   - GKE_CLUSTER_ZONE
+#   - GKE_CRED_JSON
+#   - GKE_PROJECT
+#
+#   - BACKEND
+#   - CONFIG_OVERRIDE
+#   - ENABLE_EIRINI
+#   - KUBECFG
+#   - QUIET_OUTPUT
+#   - SCF_CHART
+#   - SCF_OPERATOR
+#
+# Variables used by kubectl, gcloud, ...
+#   - KUBECONFIG
+
 set -e
+set -x
```

Comment:
Do we want `-x` here (considering secrets like `$GKE_KEY`)?

Also, if we do, please consider `set -o xtrace` (and `set -o errexit` for `-e`)?

Reply:
Correct, not anymore.

Reply:
Done with commit 54507d8b51b33626385ae6769c4abdc6b8edcd11